lib/engine/g_1822/depot.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative '../depot'
+
+module Engine
```

Comment:
avoid creating sub classes of base models, switch the depot to call an overridable method on game